scripts/start.py

```diff
@@ -52,6 +52,10 @@
     '--enable_console',
     help='optional; if specified, enables console.',
     action='store_true')
+_PARSER.add_argument(
+    '--disable_host_checking',
+    help='optional; if specified, disables host checking.',
```

Comment:
Can you give a much longer description of this, please? Otherwise no one will know what it means.

Maybe provide a link to the relevant documentation.

Reply:
I couldn't find this flag in the google cloud docs (it only listed some common flags and not the full list). When you visit the dev server without setting this flag, the following message is shown:
![image](https://user-images.githubusercontent.com/22347970/87250629-cb5f4600-c483-11ea-86f4-c4f40f26b6a9.png)

That's where I found out about this.

Reply:
Done, added more details in the text.